include/openenclave/internal/sgxcreate.h

```diff
@@ -42,7 +42,8 @@ struct _oe_sgx_load_context
     oe_sgx_load_state_t state;
 
     /* OE_FLAG bits to be applied to the enclave such as debug */
-    uint64_t attributes;
+    /* Also need xfrm to be propagated to properties */
```

Comment:
This is a non-obvious comment because you're referring to "properties" specifically as the variable used in `oe_sgx_build_enclave` but when read without that context doesn't really make sense. I suggest:

```c
/* attributes includes:
 * - OE_FLAG bits to be applied to the enclave, such as debug.
 * - XFRM supported by the OS to be used in enclave creation. 
 */
```

Reply:
Done.